src/main/content/antora_ui/src/sass/nav.scss

```diff
@@ -383,6 +385,7 @@ html.is-clipped--nav {
   border-radius: 7px;
   border-top: none;
   z-index: 2;
+  height: 75%;
```

Comment:
Is this change necessary? It does not pertain to the problem area.

Reply:
this file is added from drop-down version scroller, we don't need this file